core/trino-main/src/test/java/io/trino/sql/query/QueryAssertions.java

```diff
@@ -62,6 +63,8 @@
 {
     private final QueryRunner runner;
 
+    public static final Condition<MaterializedResult> ZERO_ROWS = new Condition<>(result -> result.getRowCount() == 0, "zero rows");
+
```

Comment:
Why not use `QueryAssertions.returnsEmptyResult()`?

Reply:
Because I forgot we had it? 😬